model/src/main/kotlin/config/ProjectExclude.kt

```diff
@@ -33,12 +33,8 @@ data class ProjectExclude(
         @JsonInclude(JsonInclude.Include.NON_NULL)
         val comment: String?,
         @JsonInclude(JsonInclude.Include.NON_EMPTY)
-        val packages: List<PackageExclude> = emptyList(),
-        @JsonInclude(JsonInclude.Include.NON_EMPTY)
-        val scopes: List<ScopeExclude> = emptyList(),
-        @JsonInclude(JsonInclude.Include.NON_EMPTY)
-        val errors: List<ErrorExclude> = emptyList()
+        val scopes: List<ScopeExclude> = emptyList()
 ) {
     @JsonIgnore
-    val exclude = packages.isEmpty() && scopes.isEmpty() && errors.isEmpty()
+    val exclude = scopes.isEmpty()
```

Comment:
Maybe comment: "Indicates whether the whole project is excluded, or whether excludes are limited to other project-specific entities".